### DISCUSSION TITLE
What is the essential significance of the existence of "location"?

### DISCUSSION THREAD

**Discussion by lusunday** [openTCS/opentcs#179] [Q&A] [answered]
I want to understand the essential reasons for using "location" relative to "point." I am considering expanding the "point" type to achieve the same effect as "location." Therefore, I would like to know what issues the use of "location" was originally intended to solve. I am now planning to remove "location" to simplify the basic elements.

**Comment by swltr** [comment, marked as answer]
@lusunday Locations are supposed to be used as destinations in transport orders, whereas points represent vehicle positions. A location can be a an area or a machine, and it can be linked to more than one point. I.e. there could be an area "goods out" that is represented by a location, and e.g. four points linked to this location that are valid vehicle positions for delivering something to this area; the dispatcher/router are free to choose any of these points as the actual destination position for the vehicle processing a transport order to "goods out".
